# Quantitative PCR Assay Development for Field Differentiation and Detection of Dispersal of the Hop Powdery Mildew Fungus
## Cameron Ross1, Michele Wiseman1, Nanci Adair2, & David Gent1,2 
### Oregon State University Department of Botany and Plant Pathology1, Forage Seed & Cereal Research Unit Corvallis, OR; USDA-ARS2

# Data description

This project attempts to create a quantitative PCR (qPCR) assay capable in field detection and quantification of Podosphaera macularis, the causal agent of Hop Powdery Mildew and use this assay to understand disease dispersal. To do this a couple things must occur: a novel region must be discovered in the diseases genes and compared to similar species, a qPCR assay will have to be designed from specific primers in this region. The first step, finding a novel region will require sequence data of different candidate genes for multiple different Podosphaera spp. This means that the first dataset will be 1) nucleotide sequences and comparisons between species. The primary data will consist sequences either compiled from in house PCR reactions analyzed on Geneious or extracted from NCBI because many of these sequences have never been generated before. The second data set will be 2) nucleotide alignments generated from both NCBI and the sequences created from our lab. This allows for visual observations on genetic differences between these species. Nucleotide data will be collected through CGRB sequencing, NCBI database, and comparisons as well as primer design will be made in Geneious through software known as Primer3(proprietary software). All qPCR data will be generated through the thermocycler in the form of a normalized fluorescence graph.  Overall this project will have at least 200-400 pieces of unique data and will likey be in the single digits of data, aproximating roughly 3 to 6 GB of data dependent on if genome mining has to be done or not. Data will be stored on a cloud server and external harddrive.

The data that is a DNA sequence that is roughly 400+ nucleotides which can be located through NCBI URL: https://www.ncbi.nlm.nih.gov/nuccore/KY786366.1. This dataset can be easily acquired through an application called Geneious through a simple click and drag function.  The licensing used within NCBI is that all data is free when attribution or appropriate credit must be given to the author. The copyright section for NCBI can be found at https://www.ncbi.nlm.nih.gov/home/about/policies/#copyright under the copyright status of webpages category. With this attribution license all sharing, copying, distribution, changes, adaptations, or additional material can be legally done even as a commercial entity if the credit has been given.

# Roles and responsibilities


All data roles will be completed by the Graduate Student Researcher(GRA), Cameron Ross, with oversight from the Principal investigator (PI) and facultative research assistant (FRA) to ensure that proposed data management guidelines are being followed. All data archiving, preservation, collection, generation, organization and analysis of DNA sequences within the Software Geneious will be controlled by the GRA. The FRA will be responsible for assisting in software utilization and maintenance as needed. The PI will assist in implementation, quality/access control as well as helping with proper analysis when the Graduate Student cannot adequately complete it. No formal definition has been provided for me in these roles, but it is understandood that it is the GRA's responsibility to ensure proper use of the data for continuing researchers. Sensitive data is any form of data that could compromise the GRA's institution and participants confidentiality and bring harmful effects to either party. The only sensitive data that will have to be dealt with involves the project would be names of farms or growers in which diseases could have been collected from throughout the course of the project. To protect these entities arbitrary code names will be created such as Oregon Farm 1, 2, 3 ect.; however, depending on the extent of disease severity further code names may be created via numberr generators which exlude state locality (adopted from social sciences practices). Since the research is partially funded though the USDA and the Hop Research Council it is expected that the generated data would be freely shared with only and attribution license. Any individual would be able to access the data freely after time of publication and there are no disclosure agreements that could affect me. Additionally, there are no intellectual property issues as the datasets and assay development will be published and deposited to GenBank also known as NCBI.

# Data standards and metadata

Meta data standards used in botany specifically pathogenic organismâ€™s genes, is the NeXus files and fasta when concerning nucleotide sequences, whether they are DNA or RNA, downloaded from National Center for Biotechonology Information (NCBI) and when nucleotide sequences are being transferred directly from individuals, when the intention is to use them in Geneious or BioEdit. Unfortunately, these are not a complete standard for everyone and are not an official requirement of databases. Even though the Research Data Alliance metadata directory states that its an international standard for the storage of equipment data, it is also commonly used to store sequence data on NCBI. NeXus is appropriate because it allows individuals to use a simple program such as notepad to open and read the files. This enables individuals to examine metadata or data even if there are not utilizing a licensed program like Geneious. Additionally, Fasta files are a common standard as long as the intention is to use them for data within Geneious because they are small and easy to use but, metadata is not encoded. All files for nucleotide sequences will kept in Geneious until they can be backedup as NeXus files periodically when the assay design is being worked on as well as being backed up in a zip format that keeps all sequence quality information and comparisons that have been made(metadata). Each file will contain within the naming convention content that includes host organism, primer region, primers used, forward or reverse primer, as well as read quality (Host_Pathogen_PrimerRegion_PrimerUsed_YYYY_MM_DD). Utulizing a long naming conventions like this allows for lots of data within the name to be stored both as NeXus and fasta files. This also is possible because NeXus files can be setup to read various columns of data and pair them.  Using the extract button located in Geneious a NeXus file can be made that will contain the nucleotide sequences, host information in descriptions, as well as the naming convention provided at the end of the pervious paragraph. Once they are converted to a NeXus file they will be named according to the same naming convention with date as YYYY.MM.DD and stored on an external hard drive. They can also be added to NCBI once assay finished and redownloaded. After assay design is complete data will be stored on an ARS cloud storage, lab external harddrive,and the lab computer that contains the Geneious software.

# Storage and security

There will be using four copy strategies to maintain data so that it will not be lost due to unauthorized access, technological failure, human error or theft. There will be an external hard drive is kept at the GRA's home which will be updated weekly to ensure that too much data is not erased and is protected from any catastrophic problems that may arise at the lab. An online cloud storage that is managed by the ARS USDA will be used to protect from any physical copy damage that may occur. There will be a working physical copy kept on the computer and another external hard drive kept at the lab in a locked office in case there are any issues with data being damaged in the cloud or deleted from the computer in case it crashes. Ideally these four copies will be enough to keep data not only secure since there will be kept in locked locations as well as in a secure online storage and will be managed by the USDA IT team. Most likely the Oregon State policies will not be used but the USDA ones will be since the work is kept in a USDA building. It will be retained by the lab and kept in an online publication source and eventually GitHub as long as is possible until the website is crashed or the lab losses funding. 

Geneious will automatically backup all of its files weekly and be upload them to the internal hardrive, external harddrive as well as the ARS cloud storage assisted by the IT team. The external hardrive will  have to be manually updated monthly to ensure it is up to date by the GRA.

The following Items will be backed up
1)Geneious zip file
2)Fasta Files extracted from Geneious kept in an automated "extract" folder

# Access and data sharing

Data will be made publicly available to NCBI around September 2022 aproximately a year and a half after publication has been submitted as well as submitted with the Journal if possible. Survey data may or may not be acquired about various farms and will be encoded if it is created either through arbitrary names or a random number generator. This is the only factor that may limit availablilty to share data and the sensitive data will only be shared under a confidentiality agreement, all other coded data will be freely available under an attribution license. All sequence data will be published individually in a NeXus file to NCBI as this can be read through notepad as well as proprietary software such as Geneious. Intellectual property will not exist directly with data generated as it will not be simply be outputs from various machinery. However, intellectual property in relation to interpretation and analysis of the journal publication may exist under attribution licenses if published. Anyform of redistribution, interprative analysis, creation of derivatives, and methods will be under attribution license but if extensively used a collaboration will be asked for if and when possible. Communication will be handled through the PI as the corresponding author of the data.
 
# Archiving and preservation

The data that comes out of the qPCR project will be restricted only to attribution copyrights and will be free to be worked on by other individuals. This will become publicly available after the publication process has been concluded. The data that is created will be posted on NCBI in primarily a NeXus and fasta format. NCBI is commonly used for many forms of DNA, RNA, and protein discplines and is specific to housing many of these codes All the data that is produced in this project will be kept on a cloud system and will be backed up to a personal external hard drive and a different lab hard drive. The PI has a habit of keeping all the data generally in a print format, cloud format, and external hard drive format. He will have it preserved most likely until he leaves the ARS (20+ years) in which the data will be handed down to his predecessor in a NeXus/fasta file formats. Since NCBI will also contains the data it will most likely be preserved by them for quite some time because one of their mission goals is to keep the data indefinitely and be a repository containing a massive amount of genomic data, in this case as NeXus/fasta files. There are factors that limit the ability to share data many are related to the researchers and the industrial companies that may own or govern the industry. Individual researchers may not want to share data because the fact that no one else owns the data means that they can be the leading researcher on that subject for quite some time because they it is a tool. Additionally, there becomes issue in relationship to funding, trade secrets, propriety software or organisms. Therefore data will only be published to a repository post-journal publication. Comparisons of species specific genes will not be posted to NCBI but all individual sequences will be posted.

Repository with source code [here](https://github.com/clarallebot/GRAD521_DMPtemplate)
